doc/null_safety/null_safe_migration.md

```diff
@@ -188,6 +196,21 @@ flowchart TD
   End_Required[/"Make it <strong>required</strong>\n<code>late SomeType propName;</code>"\]
 ```
 
+### Wrapper and `connect`ed components and required props
+
+There may be some cases where you have a wrapper component or `connect`ed component that sets some required props, 
+but you get lints and runtime errors about missing props when consuming them.
+
+See [this section](../null_safety_and_required_props.md#disabling-required-prop-validation-for-certain-props)
+of the null safety and required props docs for instructions on how to handle these cases and suppress this validation.
+
+#### connect
+
+We'll be adding a codemod to help migrate the `connect` case: https://github.com/Workiva/over_react_codemod/issues/295
```

Comment:
Should there be a mention here of what the migration looks like - like they could add the disable validation line themselves too, right?

Reply:
Or maybe link to https://github.com/Workiva/over_react/blob/e7c843022475e8e42d9e44bf2722ece990ec7336/doc/null_safety_and_required_props.md#disabling-validation-use-case-connect

Reply:
just to make it clear that they don't need to wait for the codemod

Reply:
I figured that was covered in the above section, which links to the disabling validation docs section that contains the `connect` cases, but I can definitely clarify that here and maybe link more specifically

Reply:
That makes sense! I could go either way, but reading it initially, having that section start with "We'll be adding a codemod" makes it sound like they have to wait for us to do that 🤷

Reply:
Oh for sure, and I think it should be clarified; I was more over-explaining why I initially wrote it that way, and that there was technically a link.

Addressed in https://github.com/Workiva/over_react/pull/949/commits/64c00a088908bff022005ac586c6b7201f4b5eeb; let me know what you think!

Reply:
I love it! Thank you!